modules/rest-api/pages/rest-retrieve-cluster-rebalance-reason-codes.adoc

```diff
@@ -0,0 +1,97 @@
+= Retrieving Cluster Rebalance Reason Codes
+
+:description: pass:q[If the cluster requires rebalancing, then the reason(s) for the rebalance can be ascertained using the `GET /pools/default` method.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+== HTTP method and URI
+
+----
+GET /pools/default
+----
+
+[#rest-cluster-rebalance-reason-description]
+== Description
+
+Adding or removing one or more nodes from a cluster requires a rebalance for the cluster. Use this method and URI to retrieve the reason(s) for the rebalance.
+
+NOTE: The cluster only returns the reason codes if a rebalance is required.
```

Comment:
@RayOffiah I'm wondering about this...Is the purpose of this feature to find out whether a cluster needs a rebalance? Are we asking users to check for a rebalance code as a general health check on the cluster? If so, we might want to position this a little differently.